packages/devtools_extensions/lib/src/template/extension_manager.dart

```diff
@@ -58,7 +56,7 @@ class ExtensionManager {
             break;
           case DevToolsExtensionEventType.vmServiceConnection:
             final vmServiceUri = extensionEvent.data?['uri'] as String?;
-            unawaited(appManager.connectToVmService(vmServiceUri));
+            unawaited(connectToVmService(vmServiceUri));
```

Comment:
I know this `unawaited` was already here, but would be good to catch any errors in case connecting to the vm service throws one:

`unawaited(connectToVmService(vmServiceUri).catchError(...))`